tapgarden/planter.go

```diff
@@ -155,15 +155,15 @@ type FundParams struct {
 	// TODO(jhb): follow-up PR: accept a PSBT here
 }
 
-// groupSeal specifies the group witness for a seedling in a funded batch.
-type groupSeal struct {
-	GroupMember  asset.ID
-	GroupWitness wire.TxWitness
+// MemberWitness specifies the group witness for a seedling in a funded batch.
+type MemberWitness struct {
+	Member  asset.ID
+	Witness wire.TxWitness
 }
 
 // SealParams change how asset groups in a minting batch are created.
 type SealParams struct {
-	GroupWitnesses []groupSeal
+	GroupWitnesses []MemberWitness
```

Comment:
nit: remove TODO below?

Reply:
Was related to #721 but I think that won't be a fast follow 👍🏽